cmd/telegraf/telegraf_posix.go

```diff
@@ -3,10 +3,13 @@
 
 package main
 
-func run(inputFilters, outputFilters []string) {
+import "github.com/urfave/cli/v2"
+
+func (a *Telegraf) Run() error {
```

Comment:
nit: but we renamed this from agent, but it is still referenced as `a` throughout